server/svix-server/src/error.rs

```diff
@@ -92,6 +92,13 @@ impl From<DbErr> for Error {
     }
 }
 
+impl From<svix::error::Error> for Error {
+    fn from(err: svix::error::Error) -> Self {
+        tracing::error!("Error sending operational webhook: {:?}", err);
+        HttpError::internal_server_errer(None, None).into()
+    }
+}
+
```

Comment:
We don't want to propagate this error outside. We want to just log an error with what we would have sent (so we can recover it from the logs) and continue.